<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppConfigDialog</class>
 <widget class="QDialog" name="AppConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About Me</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Normal</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>481</width>
     <height>184</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Normal</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="font">
       <font>
        <family>Normal</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Normal</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Unique ID</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_uniqueId">
      <property name="font">
       <font>
        <family>Normal</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>Normal</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Generate Unique ID</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_sic">
      <property name="font">
       <font>
        <family>Normal</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="inputMask">
       <string>XXXXXXXXXXXXXXX; </string>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Normal</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Symbol ID Code</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Normal</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Messaging Port</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Normal</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Messaging Interval (ms)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_port">
      <property name="font">
       <font>
        <family>Normal</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="inputMask">
       <string>00000; </string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_interval">
      <property name="font">
       <font>
        <family>Normal</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="inputMask">
       <string>000000000000; </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AppConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AppConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
